Wireframe/css/style.css

```diff
@@ -0,0 +1,71 @@
+*{
+    text-align: center;
+    font-family: 'Courier New', Courier, monospace;
+    
+   
+}
+
+header{
+    background-color: rgba(255, 98, 0, 0.432);
+    color: white;
+    padding: 9px;
+    margin-bottom: 10px;
+}
+
+main{
+    display: grid;
+    gap:10px;
+    grid-template-columns: 1fr 1fr;
+    grid-template-rows: auto auto;
+}
+
+.one{
+    border: 2px solid #ddd;
+    grid-column: span 2;
+    padding: 20px;
+}
+
+
+
+.two{
+    border: 2px solid #ddd;
+    grid-column: span 1;
+    grid-row: span 1;
+    padding: 20px;
+}
+
+.three{
+    border: 2px solid #ddd;
+    grid-row: span 1;
+    grid-column: span 1;
+    padding: 20px;
+}
+
+
+footer{
+    background-color: rgba(255, 98, 0, 0.432);
+    color: white;
+    padding: 9px;
+    margin-top: 10px;
+}
+
+button{
+    background-color: rgb(255, 98, 0);
+    border: rgb(255, 98, 0) solid;
+    border-radius: 3px;
+    color:white;
+   
+}
+
+a{
+    color:black;
+    text-decoration: none;
+}
+
+a:hover{
+    cursor:pointer;
+}
+
+button:hover{
+    cursor:pointer;
+}
```

Comment:
One more requirement to go here:

> The page footer is fixed to the bottom of the viewport.

Setting the footer's `position` to `fixed` would be a nice way to do that!